components/TweetCard.vue

```diff
@@ -89,6 +96,12 @@ async function deleteTweet() {
 function selectBestVariant(variants: any[]): string {
   return variants?.sort((a, b) => (b.bit_rate || 0) - (a.bit_rate || 0))[0]?.url || ''
 }
+
+function handleImageError() {
+  if (avatarImage.value) {
```

Comment:
avatarImage can never be undefined, so just set `avatarImage.value` here without the `if`